### PR TITLE
Fix Teleport sounds when teleport fails (#137)

### PR DESCRIPTION
**Pull request by MoNTE48**
Resolved: #137

### CODE REVIEW

File: register.lua [hunk 1]
```diff
@@ -76,9 +76,9 @@ unified_inventory.register_button("home_gui_go", {
 	action = function(player)
 		local player_name = player:get_player_name()
 		if minetest.check_player_privs(player_name, {home=true}) then
-			minetest.sound_play("teleport",
-				{to_player=player:get_player_name(), gain = 1.0})
-			unified_inventory.go_home(player)
```
Comment (Panquesito7) [inline register.lua:81]: Why remove this?

File: register.lua [hunk 2]
```diff
@@ -76,9 +76,9 @@ unified_inventory.register_button("home_gui_go", {
...
+			if unified_inventory.go_home(player) then
+				minetest.sound_play("teleport", {to_player = player_name})
```
Comment (Panquesito7) [inline register.lua:80]: Preferably, use the same as-is implementation.

File: register.lua (hunk 1 above)
Comment (MoNTE48) [reply register.lua:81]: what exactly?

Comment (Panquesito7) [reply register.lua:81]: The `unified_inventory.go_home(player)` code.

Comment (MoNTE48) [reply register.lua:81]: Check line 79

File: register.lua (hunk 2 above)
Comment (MoNTE48) [reply register.lua:80]: Please look carefully at my code.

**Pull request merged**